hub/config/src/test/resources/remote-test-config.json

```
@@ -1 +1,54 @@
+{
+  "published_at":"2019-05-16T14:35:52.991+00:00",
+  "event_id":132,
+  "connected_services" : [
+    {
+      "entity_id" : "https://bananaregistry.service.gov.uk",
+      "matching_service_adapter" : "https://msa.bananaregistry.service.gov.uk",
+      "service_provider": 2
```

```suggestion
      "service_provider_id": 2
```
so it's clearer it's just a reference, not the actual service_provider object